extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -289,12 +307,18 @@ export class AmpLiveList extends AMP.BaseElement {
   /**
    * Record ids of previously seen children to cache.
    *
-   * @param {string} id
-   * @param {?string} updateTime
+   * @param {!Element} child
    * @private
    */
-  cacheChildId_(id, updateTime) {
-    this.knownChildIds_[id] = updateTime;
+  cacheChild_(child) {
+    const id = child.getAttribute('id');
+    const updateTime = getNumberMaxOrDefault(
```

Comment:
Suggestion: Make `getUpdateTime` encapsulate the fallback to sort time. I think that value can be used throughout. The current fallback to 0 can just go away I think.